docs/configure/automation/integrations.md

```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
   body: JSON.stringify(ActionItem),
 });
 ```
+
+## Adding and Using Secrets
+Users can store encrypted secrets into Insights and use them when creating Automation Rules using the `getSecret` function. 
+
+Create a secret:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/$YOUR_ORG/secrets/bulk' \
+  -H 'accept: application/json, text/plain, */*' \
+  -H 'content-type: application/json' \
+  ...
+    --data-raw '[{"key": "$KEY", "value": "$VALUE"}]'
+```
+
+Listing secrets keys endpoint:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/acme-co/secrets/bulk'
+...
```

Comment:
Not sure what the ... is? I think it should be removed

Reply:
Cookies and other http parameters

Reply:
I would remove it then